algorithms/48/fib.cpp

```diff
@@ -1,17 +1,16 @@
 #include <iostream>
 using namespace std;
 
-// returns the nth fibonacci number
-long long fib(int n) {
-    if(n == 0) return 0;
-    if(n == 1) return 1;
-    return fib(n-1)+fib(n-2);
-}
 
 int main() {
     int n;
     cout << "Enter a number: ";
     cin >> n;
-    cout << "The " << n << "th fibonacci number is " << fib(n) << endl;
+	long long fib[n+1];
```

Comment:
Do you need to store all the fibonacci numbers less than the ```nth``` fibonacci number to compute it?

Reply:
This was done to reduce computational load